app/containers/ProblemsPage/index.js

```diff
@@ -79,13 +85,22 @@ export default class ProblemsPage extends React.PureComponent {
 
     const isAdmin = contest && contest.owners.map((o) => o.id).includes(this.props.user.id);
 
+    const submitMetrics = contest.submitMetrics ?
+      Map(contest.submitMetrics.map((metric) => [metric.status, metric.submits])) : Map();
+
     return (<div>
       {isAdmin && <Paper style={{ margin: 16, padding: '6px 10px 6px 0' }}>
         <Grid container align="center" spacing={0} justify="space-between">
           <Link to={contestResults(contestId)}><Button><FormattedMessage {...messages.results} /></Button></Link>
-          <Typography type="body1"><FormattedMessage {...messages.submited} values={{ number: 100 }} /></Typography>
-          <Typography type="body1"><FormattedMessage {...messages.valid} values={{ number: 50 }} /></Typography>
-          <Typography type="body1"><FormattedMessage {...messages.timeout} values={{ number: 10 }} /></Typography>
+          <Typography type="body1">
+            <FormattedMessage {...messages.submited} values={{ number: submitMetrics.reduce((s, v) => s + v) }} />
```

Comment:
`submitMetrics.reduce((s, v) => s + v)` nazwałbym to żeby było wiadomo o co chodzi